source/general/wikicontributing.rst

```diff
@@ -4,64 +4,66 @@
+.. note:: When installing Python make sure to tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
 
-First thing you need to do is to fork the git repository and then clone it.
+
+1. Fork the GitHub repository and clone it.
 
 .. note:: Remember to replace *exampleusername* with your actual github username
 
 .. code:: bash
 
     git clone https://github.com/exampleusername/ProjectKorraWiki.git
 
-The wiki is made with sphinx using the *sphinx_rtd_theme* from readthedocs. We've bundled all you need into one file.
-Navigate to the directory you cloned the project to, then we will install all the requirements by running the following command in terminal / command prompt
+The wiki is built with Sphinx using the *sphinx_rtd_theme* from ReadTheDocs.
+Navigate to the directory you cloned the project to, then it will install all the requirements by running the following command in terminal / command prompt.
 
 .. code:: bash
 
     pip install -r .\requirements.txt
 
-This should install both sphinx as well as the theme.
+This should install both Sphinx as well as the theme.
 
-To check that sphinx is correctly install run:
+To check that Sphinx is correctly installed, run:
 
 .. code:: bash
 
     sphinx-build -version
 
-If sphinx has been installed correctly it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
+If Sphinx has been installed correctly, it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
 
-The reason we need sphinx is so you can view your changes locally before you submit your changes as a pull request.
+Sphinx is required to view your changes locally before submitting them.
 
-Viewing your changes locally
-============================
+Viewing Local Changes
+=====================
 
-Once you have made some changes and you are ready to see your edits locally open terminal / command prompt and run
+Once you have made some changes and are ready to see your edits locally, open terminal / command prompt and run
 
 .. code:: bash
 
     .\make.bat html
 
-This should build a copy of the wiki into the build folder in the root directory of your repository.
-To view the wiki double click on ``build > html > index.html``
+This should build an HTML version of the wiki in the build folder in the root directory of your repository under the html folder.
 
-Submitting your changes
-=======================
+Submitting Changes
+==================
 
 Now that you are happy with your changes remember to commit them using git and make sure to put a detailed message indicating what you have done.
-If you are fixing/completing an issue please also reference the issue number in your commit message.
+If you are working on an issue, please include a link to the corresponding Trello Card/GitHub Issue in your commit description.
```

Comment:
Change the `GitHub Issue / Trello Card` to keep consistency, you add spaces between all other slashes.